docs/src/examples/components/SplitButton/Types/SplitButtonExample.shorthand.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react'
+import { Flex, SplitButton } from '@stardust-ui/react'
+
+class SplitButtonExampleShorthand extends React.Component {
```

Comment:
It is possible to get into this state using keyboard:
![image](https://user-images.githubusercontent.com/10930160/64123517-dd248700-cda4-11e9-908f-a7ad9d8ddf7b.png)

When using mouse only, only one option menu can be open at the same time, which I think it correct